5-fetching-and-mutating-data/bee-rich/README.md

```diff
@@ -293,10 +293,10 @@ seed();
 This file will seed the database with some sample data. We can execute the script by running the following command:
 
 ```bash
-npx ts-node prisma/seed.ts
+npm run seed
```

Comment:
This might be the only "controversial" line, so do with it what you will. I just thought it a bit confusing to not use the command you already defined in our `scripts` portion of `package.json`

Reply:
Agreed! Thanks for bringing this up! :)